src/util/convert_xml_json.h

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016-2020 Diffblue Limited. All Rights Reserved.
+
+#ifndef CPROVER__CONVERT_XML_JSON_H
+#define CPROVER__CONVERT_XML_JSON_H
+
+#include "json.h"
+#include "xml.h"
+#include <algorithm>
+json_objectt from_xml(xmlt data);
+
+std::string xml_to_pretty(const xmlt &data);
```

Comment:
Given that `xml_to_pretty` isn't outputting json, I'd be tempted to call this new module something slightly different, e.g. `xml_conversion.{cpp,h}` or `xml_output_conversion.{cpp,h}` or something similar? Also, given that both these two functions are (at a high level) "reproduce XML in some other output format", why are the names of the two so different, and why does one take a `const xmlt &`, but the other takes a plain `xmlt` ? I'd ideally like to see both of these as `const xmlt &` inputs.

Reply:
Resolved by removing this file